nanovg.cabal

```diff
@@ -25,6 +25,11 @@ flag examples
   default:     False
   manual:      True
 
+flag gles3
```

Comment:
It looks like this also disables the `GL3` build so that should be mentioned in the flag description.